config/salt/minion.d-ca/signing_policies.conf

```diff
@@ -3,8 +3,8 @@ x509_signing_policies:
     - minions: '*'
     - signing_private_key: /etc/pki/private/ca.key
     - signing_cert: /etc/pki/ca.crt
-    - basicConstraints: "CA:false"
-    - keyUsage: "nonRepudiation, digitalSignature, keyEncipherment"
+    - basicConstraints: "critical CA:false"
+    - keyUsage: "critical nonRepudiation, digitalSignature, keyEncipherment"
```

Comment:
Not sure, is there a reason for this not to be comma-separated? I know it wasn't comma separated before, just asking if this has any impact at all, however comma separated or not I think it would be good to have both `keyUsage` in the same way.